### ISSUE TITLE
loss为0

### ISSUE THREAD

**Issue by Richard12868** [THUDM/GLM-4#260]
### System Info / 系統信息

系统信息完全默认，使用给定的数据，loss一直为0
{"messages": [{"role": "user", "content": "类型#裤*材质#牛仔布*风格#性感"}, {"role": "assistant", "content": "3x1的这款牛仔裤采用浅白的牛仔面料为裤身材质，其柔然的手感和细腻的质地，在穿着舒适的同时，透露着清纯甜美的个性气质。除此之外，流畅的裤身剪裁将性感的腿部曲线彰显的淋漓尽致，不失为一款随性出街的必备单品。"}]}

### Information / 问题信息

- [X] The official example scripts / 官方的示例脚本
- [ ] My own modified scripts / 我自己修改的脚本和任务

### Reproduction / 复现过程

***** Running training *****
  Num examples = 1,010
  Num Epochs = 24
  Instantaneous batch size per device = 1
  Total train batch size (w. parallel, distributed & accumulation) = 8
  Gradient Accumulation steps = 1
  Total optimization steps = 3,000
  Number of trainable parameters = 2,785,280
{'loss': 0.0, 'grad_norm': 0.0, 'learning_rate': 0.0004996666666666667, 'epoch': 0.02}
{'loss': 0.0, 'grad_norm': 0.0, 'learning_rate': 0.0004993333333333334, 'epoch': 0.03}
{'loss': 0.0, 'grad_norm': 0.0, 'learning_rate': 0.000499, 'epoch': 0.05}
{'loss': 0.0, 'grad_norm': 0.0, 'learning_rate': 0.0004986666666666667, 'epoch': 0.06}
{'loss': 0.0, 'grad_norm': 0.0, 'learning_rate': 0.0004983333333333334, 'epoch': 0.08}
{'loss': 0.0, 'grad_norm': 0.0, 'learning_rate': 0.000498, 'epoch': 0.09}

### Expected behavior / 期待表现

loss有值

**Comment by hecong97** [comment]
hello，请问这个问题是什么原因呢，我也遇到了

**Comment by WangZhuoxian** [comment]
遇到了一样的问题，排查了不是max_len的设置问题，也不是finetune.py需要改啥的问题，而是deepseed的问题，放弃用zero2改用zero3就行，或者按[这里](https://github.com/microsoft/DeepSpeed/issues/5242#issuecomment-2105624079)改设置也行
